public/js/humanize.js

```diff
@@ -8,8 +8,18 @@ var humanize = (function() {
     return Math.round(value * multiplier) / multiplier
   }
   return {
-    decimalParts: function(v, useCommas, precision, asString) {
-      if (!precision || precision > 8) {
+    fmtPercentage: function(val) {
+      var sign = "+"
+      var color = "green"
+      if (val < 0 ){
+        sign = ""
```

Comment:
Should this be `"-"`?

Reply:
And 0 is technically not + or -, but that's no biggie.

Reply:
ok. Noted

Reply:
If a value is less than 0 it already has a negative sign.